doc/flatpak-remote-info.xml

```diff
@@ -101,6 +101,16 @@
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--cached</option></term>
+
+                <listitem><para>
+                  Prefer to use locally cached information if possible, even though it
```

Comment:
Is the 'if possible' misleading here? Don't we just fail if the data is not available?

Reply:
No, the CLI does a fallback if it returns ERROR_NOT_CACHED. I was thinking we could do that in the API too, but that is a less generic api. I.e. you can implement prefer-cached on top of only-from-cache, but not the other way around.